<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterEntraineur</class>
 <widget class="QWidget" name="AjouterEntraineur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="text_specialite">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de titres</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_prenom">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>150</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Annuler">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons8_back_to_200px.png</normaloff>:/icons/icons8_back_to_200px.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="Valider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/icons8_checkmark_96px.png</normaloff>:/icons/icons8_checkmark_96px.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nueva Std</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajouter un nouveau entraineur</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="text_nom">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenom">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titres">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="specialite">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;


	selection-color: rgb(60, 63, 65);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Specialit√©</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_titres">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
